wiki/Storyboard/Scripting/en.md

```diff
@@ -1,34 +1,39 @@
 # Storyboard scripting
 
-![An example of scripting in .osb](img/SBS_Base.jpg "An example of scripting in .osb")
+![Screenshot of a storyboard script open in "Code - OSS"](img/osb-example.png "Example of a storyboard script (.osb)")
 
-In addition to the [built-in storyboard editor](/wiki/Client/Beatmap_editor/Design), a plain text editor, such as Notepad, [Notepad++](https://www.notepad-plus-plus.org/), or [Visual Studio Code](https://code.visualstudio.com/), can also be used to create [storyboards](/wiki/Storyboard) for beatmaps. This workflow is called **storyboarding by scripting** (*SBS*) and it allows one to be more precise (to the pixel and the millisecond) as to when sprites appear, how they behave, and so on.
+**Storyboard scripting** is the process of editing osu! [storyboards](/wiki/Storyboard) via their `.osb` and `.osu` files. These files define the images and effects that create  storyboard videos in-game.
```

Comment:
videos are not storyboards

```suggestion
**Storyboard scripting** is the process of editing the [`.osb`](/wiki/Client/File_formats/Osb_(file_format)) and [`.osu`](/wiki/Client/File_formats/Osu_(file_format)) files of a [beatmap](/wiki/Beatmap). These files define images and effects that create background animations during gameplay, called "[storyboards](/wiki/Storyboard)".
```

Reply:
however, videos can be classified and modified as a storyboard object.

Reply:
yes, they are the same enum in the code, but they have different tags on the website ([most of the time](https://github.com/ppy/osu-web/issues/7330)), are functionally different and require different workflows. they aren't [listed as storyboard objects on the wiki](https://osu.ppy.sh/wiki/en/Storyboard/Scripting/Objects) and you can't put commands on them[*](https://discord.com/channels/203050773645492224/484002829728743424/922512230925369374)

Reply:
its just language im not talking about the Video objects, but ya both "video" and "animation" are unfortunately used in storyboard context and i couldnt think of any other words...

i dont think the description needs to be attributed with "called storyboard" either, or at least sounds weird to me considering the page is a sub-topic of storyboard

will add the links & try to apply general idea tho, i agree "video" could be misleading

Reply:
juggled some words around here